readthedocs/projects/urls/private.py

```diff
@@ -70,7 +70,10 @@
     ),
     path(
         "import/manual/",
-        ImportWizardView.as_view(),
+        ImportWizardView.as_view(
+            ImportWizardView.form_list,
+            condition_dict={"config": ImportWizardView.show_config_step},
```

Comment:
You should be able to declare this as a class attribute instead of passing it to the view.

Reply:
I used this pattern because it's the one mentioned in the documentation.

Reply:
Yeah, but it's confusing having this logic outside the class.

Reply:
I had to move the method outside the class.